docs/semgrep-cloud-platform/github-pr-comments.md

```diff
@@ -36,7 +36,7 @@ Semgrep can create **pull request (PR) comments** in your GitHub repository. The
 - You must connect your GitHub organization (org) to Semgrep.
     - For **GitHub Cloud**: See [<i class="fa-regular fa-file-lines"></i> Adding an organization](/semgrep-cloud-platform/getting-started/#adding-an-organization).
     - For **GitHub Enterprise Server**: [<i class="fa-regular fa-file-lines"></i> Integrate Semgrep with GitHub Enterprise](/semgrep-cloud-platform/scm/#integrating-semgrep-cloud-platform-with-github-enterprise-or-gitlab-self-managed) and if you have a VPN or firewall, [set up your allowlists](/semgrep-cloud-platform/scm/#receiving-pr-or-mr-comments-in-your-vpn-or-on-premise-scm).
-- You must add or onboard a Semgrep project (repository) to SCP and it must complete at least one full scan on your default branch successfully.
+- You must add or onboard a Semgrep project (repository) to SCP. Semgrep recommends that a project first complete at least one full scan on your default branch successfully.
```

Comment:
@s-santillan So far there isn't really a firm indication that lacking a full scan would cause this problem. You can check on [FIND-1441](https://linear.app/semgrep/issue/FIND-1441/reports-of-differential-pr-scans-on-projects-without-full-scans) (internal link) to see the details of the conversation. Given that I'm going to leave this sentence as-is for now.

Reply:
Thanks for the link! Very helpful.